frontend/src/app/main/ui/workspace/sidebar/options/utils.cljs

```diff
@@ -0,0 +1,7 @@
+(ns app.main.ui.workspace.sidebar.options.utils
+  (:require [app.main.data.workspace :as dw]
+            [app.main.store :as st]))
+
+(defn trigger-bb-cloaking! [ids]
+  (when (seq ids)
+    (st/emit! (dw/trigger-bounding-box-cloaking ids))))
```

Comment:
I'm not clearly understand this function, you can emit the event directly on all the sites where `trigger-bb-cloaking!` helper is used, the difference is 1 line,  and that additional line can be avoided by just making the check in the event.

I suggest emitting event directly on the handlers.